config/hibp.spc

```diff
@@ -1,4 +1,8 @@
 connection "hibp" {
-  plugin = "wedtm/hibp"
-  api_key = "replace-me-with-your-own-key"
-}
\ No newline at end of file
+  plugin  = "hibp"
+  
+  # Requests to HIBP API needs to carry an API KEY.
+  # You can get one at https://haveibeenpwned.com/API/Key
+  # These can also be 'HIBP_API_KEY'
```

Comment:
This sentence feels incomplete, can you please use the same wording from https://github.com/turbot/steampipe-plugin-mongodbatlas/blob/main/config/mongodbatlas.spc#L7-L9?